sprimber-cucumber/src/main/java/com/griddynamics/qa/sprimber/stepdefinition/CucumberTestMethodFactory.java

```diff
@@ -105,7 +113,11 @@ private TestMethod buildHookMethod(Method method, Annotation annotation) {
         testMethod.addAttribute(ORDER_ATTRIBUTE, AnnotationUtils.getValue(annotation, ORDER_ATTRIBUTE.toLowerCase()));
         testMethod.addAttribute(TIMEOUT_ATTRIBUTE, AnnotationUtils.getValue(annotation, TIMEOUT_ATTRIBUTE.toLowerCase()));
         if (!Arrays.asList(AnnotationUtils.getValue(annotation)).isEmpty()) {
-            testMethod.addAttribute(TAGS_ATTRIBUTE, StringUtils.join((String[]) AnnotationUtils.getValue(annotation), ','));
+            if (AnnotationUtils.getValue(annotation) instanceof String) {
+                testMethod.addAttribute(TAGS_ATTRIBUTE, AnnotationUtils.getValue(annotation));
```

Comment:
Looks like some workaround. Maybe there is some more native way to avoid if statements?

Reply:
yeap, workaround to support different versions of Cucumber annotations. will be fixed once the old annotation will be removed